### DISCUSSION TITLE
Boilerplate Vercel Project.

### DISCUSSION THREAD

**Discussion by BrandonCopley** [vercel/next.js#19534] [Help] [answered]
I'm getting this error when running vercel dev on a new server Port 4000 is already in use.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
^Cinternal/process/per_thread.js:196
      throw errnoException(err, 'kill');

Port 4000 is the port I'm trying to run on. Any ideas?

**Comment by kachkaev** [comment, marked as answer]
It is likely that you have started a dev server of some sort in a different Terminal tab but forgot to kill it. You can use [`kill-port`](https://www.npmjs.com/package/kill-port) as a helper tool to kill a process that is using the port you need:

```sh
npx kill-port 4000
```

This command must be used with caution, after checking what exactly is listening on the port. Stopping an important process unintentionally may be harmful.

---

Note that by default Next.js is using port 3000, not 4000. That suggests that you are running something bespoke. If it was a standard Next.js app, you could try `npm run dev -p 4242` to run your app on port 4242 instead of 3000. You can use any port that’s not used by other applications. Max port number is 65535.

**Reply by BrandonCopley** [reply]
It appears an internal package by google was spinning up another server on the same port. Google oauth opens its own server.

**Comment by BrandonCopley** [comment]
Closed
